app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -52,6 +52,15 @@
 
 (def milliseconds-when-resent (* 1000 5))                  ;; 5 seconds
 
+(defn- modified-since-last-seen
+  "Show new indicator icon if entity has changed since last seen."
+  [seen-at {:meta/keys [created-at modified-at]}]
+  (let [changed-at (or modified-at created-at)]
+    (when (and seen-at changed-at
+               (> (.getTime changed-at) (.getTime seen-at)))
+      [icons/alert-error-outline {:class "new-indicator"
+                                  :style {:color theme-colors/orange}}])))
```

Comment:
We now have at least comments and meetings with this kind of functionality. If we get more in the future, it might be worthwhile to come up with a general approach to handling "you haven't seen this / this has changed since you've seen it", both in the application logic and UI level. Just something to keep in mind.